src/installer/tests/HostActivation.Tests/DependencyResolution/RidAssetResolution.cs

```diff
@@ -40,102 +34,113 @@ protected RidAssetResolutionBase(SharedTestState sharedState)
         [InlineData("win10-x64", WindowsAssembly, $"{LinuxAssembly};{MacOSAssembly}")]
         [InlineData("linux-x64", LinuxAssembly, $"{MacOSAssembly};{WindowsAssembly}")]
         [InlineData("osx-x64", MacOSAssembly, $"{LinuxAssembly};{WindowsAssembly}")]
-        public void RidSpecificAssembly(string rid, string includedPath, string excludedPath)
+        public void RidSpecificAssembly(string rid, string includedPath, string excludedPath, bool hasRuntimeFallbacks = true)
```

Comment:
Subjective - Presonally I would prefer to add a new method overload and call the "private" one from other places. I just don't like that a method is both a test and also called from another test.